### PR TITLE
bpo-30940: fix docs for round if second arg is None

### PR DESCRIPTION
**Pull request by daxlab**
https://bugs.python.org/issue30940

### CODE REVIEW

File: Doc/library/functions.rst [hunk 1]
```diff
@@ -1308,8 +1308,8 @@ are always available.  They are listed here in alphabetical order.
    equally close, rounding is done toward the even choice (so, for example,
    both ``round(0.5)`` and ``round(-0.5)`` are ``0``, and ``round(1.5)`` is
    ``2``).  Any integer value is valid for *ndigits* (positive, zero, or
-   negative).  The return value is an integer if called with one argument,
-   otherwise of the same type as *number*.
+   negative).  The return value is an integer if *ndigits* is omitted or *None*.
```
Comment (serhiy-storchaka) [inline Doc/library/functions.rst]: Format `None` with fixed-width font instead of italic.

**Review by serhiy-storchaka — CHANGES_REQUESTED** [review]
Needed to update the docstring too.

The signature of the function can be written as
```
round(number, ndigits=None)
```

**Review by mdickinson — APPROVED** [review]
LGTM. Thanks for doing this!

**Comment by serhiy-storchaka** [comment]
Please update the docstring.

And what is more correct,
```
if *ndigits* is omitted or is ``None``
```
(as in the first paragraph), or
```
if *ndigits* is omitted or ``None``
```
(as in the second paragraph)? Or both are correct?

**Comment by daxlab** [comment]
`if *ndigits* is omitted or ``None`` ` looks correct.

File: Python/bltinmodule.c [hunk 2]
```diff
@@ -2086,7 +2086,7 @@ builtin_round(PyObject *self, PyObject *args, PyObject *kwds)
 }
 
 PyDoc_STRVAR(round_doc,
-"round(number[, ndigits]) -> number\n\
+"round(number, ndigits=None) -> number\n\
 \n\
 Round a number to a given precision in decimal digits (default 0 digits).\n\
 This returns an int when called with one argument, otherwise the\n\
```
Comment (serhiy-storchaka) [inline Python/bltinmodule.c]: "if ndigits is omitted or None"

**Comment by mdickinson** [comment]
About

> if *ndigits* is omitted or is ``None``

versus

> if *ndigits* is omitted or ``None``

I think either is fine. To my eyes, the first looks a bit clearer, but also a bit more pedantic.

**Comment by daxlab** [comment]
@serhiy-storchaka docs updated.

**Comment by daxlab** [comment]
@serhiy-storchaka plz review. Also, should I send a separate PR to 3.6 branch or you can cherry-pick ?

**Review by serhiy-storchaka — APPROVED** [review]

**Comment by serhiy-storchaka** [comment]
The current changes LGTM, but see a discussion on the tracker. Perhaps the documentation needs additional clarification.

**Pull request closed without merging**